e2e/multi-svc-app/multi_svc_app_test.go

```diff
@@ -256,6 +298,19 @@ var _ = Describe("Multiple Service App", func() {
 			bodyBytes, err := ioutil.ReadAll(resp.Body)
 			Expect(err).NotTo(HaveOccurred())
 			Expect(string(bodyBytes)).To(Equal("back-end-service-discovery"))
+
+			// Job should have run.
+			Eventually(func() (string, error) {
```

Comment:
nit: is it possible to move this test to its own `It` block so that it's easier to identify if a failure is related to the job functionality